src/style.css

```diff
@@ -0,0 +1,57 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  width: 100px;
+  height: 100px;
+}
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+  padding-top: 2px;
+}
+
+.stars--0 {
+  transform: translateY(2px);
+}
+
+.stars--1 {
+  transform: translateY(2px);
+}
+
+.stars--2 {
+  transform: translateY(2px);
+}
+
+.stars--3 {
+  transform: translateY(2px);
+}
+
+.stars--4 {
+  transform: translateY(2px);
+}
+
+.stars--5 {
+  transform: translateY(2px);
+}
+
+.stars__stars {
+  height: 16px;
+  width: 16px;
+  background-image: url("images/star.svg");
+  background-repeat: no-repeat;
+  margin: 0 2px;
+}
+
+.stars-star {
+  height: 16px;
+  width: 16px;
+  background-image: url("images/star-active.svg");
+  background-repeat: no-repeat;
+  margin: 0 2px;
+}
```

Comment:
```suggestion
.stars--1 :first-child,
.stars--2 :nth-child... {
  background-image: url("images/star-active.svg");
}
```

Reply:
just change bg image for nth-child when stars has modifier with number of active stars
(add for all numbers)